package.json

```diff
@@ -50,7 +50,12 @@
 		"typedoc-theme-hierarchy": "^4.0.0"
 	},
 	"dependencies": {
-		"@polkadot/api": "^14.0.1",
+		"@polkadot/api": "14.0.1",
```

Comment:
Newest versions are 14.1.1, and 13.2.1

Reply:
Ill let dependabot update it. Trying to avoid changing the lockfile b.c i think updating will break the code

Reply:
Makes sense, thanks!